docs/operator/k8s_guide.md

```diff
@@ -0,0 +1,725 @@
+
+Then `kubeseal` perform the encryption and the encoding, saving the secret to a file stored in `SECRETS_STORE`.
+Finally, `kubectl apply` applies the secret to the cluster. Please notice that is eventually also achieved via flux, by pushing the secret file to the repository.
+
+:::tip[Secret tip]
+IF `kubeseal` returns an error, it's probably because it's not installed:
+```sh
+wget https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.18.0/kubeseal-0.18.0-linux-amd64.tar.gz
+tar xfz kubeseal-0.18.0-linux-amd64.tar.gz
+sudo install -m 755 kubeseal /usr/local/bin/kubeseal
+```
+Adjust the version and the build details according to your machine.
+:::
+The content of the secret will be of this type: 
+```yaml ss_rucio-servers-hostcert.yaml
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: rucio-servers-hostcert
+  namespace: rucio
+spec:
+  encryptedData:
+    hostcert.pem: <encrypted secret>
+  template:
+    data: null
+    metadata:
+      creationTimestamp: null
+      name: rucio-servers-hostcert
+      namespace: rucio
+```
+## Rucio Servers
+[Reference Helm chart](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/sync/rucio-servers.yaml?ref_type=heads).
+
+>The Rucio servers are the backbone of the Rucio system. They handle all core functionalities including data management, rule-based data replication, data placement, and monitoring. The servers ensure the integrity and availability of data across various storage systems.
+
+In order to setup this service, four steps are needed:
+1. Produce the Helm chart
+2. Setup the LB
+3. Produce the certificates related to the landb-alias used
+4. Add the certificates as secrets and mount them on the k8s pods.
+
+### Produce the Helm chart
+Please look at the [currently used one](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/sync/rucio-servers.yaml?ref_type=heads).
+A few remarks:
+
+- The db secret is being mounted on `config.database.default` in order to allow the user to correctly access it.
+- `hostcert.pem`, `hostkey.pem` and `ca.pem` are automatically detected, so they don't need to be mounted. Nevertheless, the corresponding secrets must be created (see next sections).
+- The GridCA cert is needed to verify all the other certs. Please look at this [git commit](https://gitlab.cern.ch/rucio-it/flux-compass/-/commit/688da7285833dafd1bbe25469f35c6059d7af24f) used to set the `GridCA` file in `/etc/grid-security/certificate`.
+	- Together with this, is also necessary to setup the correspoding variable `RUCIO_CA_PATH`
+- The `RUCIO_SSL_PROTOCOL` variable must be explicitly set.
+
+Please notice that in order to have LBs produced, ***the Helm chart must be applied***. 
+This will come with several errors related to certificates, that will be fixed in the next steps.
+### Setup the LB
+The minimal configuration for LBaaS is the following:
```

Comment:
Undefined acronym. Where is this config used? Is it included in the helm chart?

Reply:
Acronym is introduced [here](https://github.com/rucio/documentation/blob/74c58b5a8cb2ca26d1863443343d17013e53a9ca/docs/operator/k8s_guide.md#creating-a-landb-set). I can re-iterate it also here if it helps the user.

Reply:
I think that's a non-trivial jump in logic, that's a good clarification to include.

Reply:
Added:
>With respect to the [original Helm chart](https://github.com/rucio/helm-charts/blob/7f8a7d9fb9cbcd01d645f24523a173c7f53fb101/charts/rucio-server/values.yaml#L33), that employs the Kubernetes native [`ClusterIP`](https://kubernetes.io/docs/concepts/services-networking/cluster-ip-allocation/) service, this configuration exploits the CERN native LBaaS, and exposes it over HTTPS.